src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -222,6 +225,19 @@ private void parseEntityJson(String entityJson) throws InvalidInputException {
     }
   }
 
+  /**
+   * For each custom entity found in the entityJson payload we:
+   * - create a BiItem containing the type of entity found
+   * - increase the total count of entities found in the message
+   */
+  private void addCustomEntitiesToBiContext(JsonNode entityNode) {
+    entityNode.findValues("type").forEach(entityType -> {
```

Comment:
Is  "type" the same as Entity.TYPE_FIELD?

Reply:
yes let me reuse this constant